runtime/src/accounts_db.rs

```diff
@@ -4020,16 +4005,12 @@ impl AccountsDB {
     pub fn get_accounts_delta_hash(&self, slot: Slot) -> Hash {
         let mut scan = Measure::start("scan");
 
-        let scan_result: ScanStorageResult<(Pubkey, Hash, u64), DashMapVersionHash> = self
+        let scan_result: ScanStorageResult<(Pubkey, Hash), DashMapVersionHash> = self
             .scan_account_storage(
                 slot,
                 |loaded_account: LoadedAccount| {
                     // Cache only has one version per key, don't need to worry about versioning
-                    Some((
-                        *loaded_account.pubkey(),
-                        *loaded_account.loaded_hash(),
-                        CACHE_VIRTUAL_WRITE_VERSION,
```

Comment:
@carllin I think this was lamports. Can you please help me verify removing this field is the correct thing to do if we're not using lamports anyway?

Reply:
looks great to me!. The lamports are always set to 0 anyways, and the call to `accumulate_account_hashes()` only returns the computed hash anyways. I've been meaning to clean this up at some point, thanks for taking it!